lib/linter-rubocop.coffee

```diff
@@ -9,7 +9,8 @@ class LinterRubocop extends Linter
 
   # A string, list, tuple or callable that returns a string, list or tuple,
   # containing the command line (with arguments) used to lint.
-  cmd: 'rubocop --force-exclusion --format emacs'
+  cmd: atom.config.get("linter-rubocop.cmd") ||
+         'rubocop --force-exclusion --format emacs'
```

Comment:
If that is the behaviour of `atom.config.get`, then you are fully right. I had a quick look on https://atom.io/docs/v0.186.0/advanced/configuration but I could not find any information about it, without diving in the code. I will remove the useless part :).

However, that was a quick fix to allow me to linter something in a very specific case. Somehow it feels like an ugly solution for the problem but I thought someone with the same issue could be interested on this.